doc/introduction.rst

```diff
@@ -107,27 +123,29 @@ Glossary: machine learning vocabulary
 Installing nilearn
 ====================
 
-Installing the Python scientific environment
+Installing the scientific Python environment
 ----------------------------------------------
 
 We recommend that you **install a complete scientific Python distribution**,
-and not download the bare Python. Indeed, the scientific Python tool stack is
-rich. Installing the different packages needed is time-consuming and error
+rather than downloading the bare Python.
+Given the richness of the scientific Python tool stack, installing
+the various required packages may be time-consuming and error
 prone.
 
 :Windows and MacOSX:
   We suggest you to install 64 bit Anaconda_.
 
   `Enthought Canopy`_, `PythonXY <http://code.google.com/p/pythonxy/>`_ are
-  other options. Enthought Canopy Express, the free version, should cover all
-  the required packages.
+  alternative options.
+  Enthought Canopy Express (download the free version) should provide all
+  the entirety of required packages.
```

Comment:
Not sure that I prefer your formulation on this one. But can't tell why, that's totally subjective.